test/ecto/migrator_test.exs

```diff
@@ -203,6 +212,18 @@ defmodule Ecto.MigratorTest do
     end
   end
 
+  test "the migration can be disabled" do
+    capture_log fn ->
+      up(MockRepo, 0, NoTransactionMigration)
+
+      # Assert there's only one transaction message, which is for when the
+      # SchemaMigration does his thing. If @disable_ddl_transaction was set to
+      # false, we would have *two* {:transaction, _} messages.
+      {:messages, messages} = Process.info(self, :messages)
```

Comment:
You could do:

```elixir
# From the SchemaMigration insert transaction
assert_received {:transaction, _}

# No transaction from migration
refute_received {:transaction, _}
```

In fact, maybe we should add a test showing that repos with DDL transaction do run two transactions?